packages/jaeger-ui/src/model/ddg/transformTracesToPaths.test.js

```diff
@@ -22,10 +22,18 @@ describe('transform traces to ddg paths', () => {
     })),
     attributes: [{ key: 'exemplar_trace_id', value: trace.data.traceID }],
   });
-  const makeSpan = (spanName, parent, kind) => ({
+
+  const addExemplarTraceIDs = (path, traces) => {
+    traces.forEach(trace => {
+      path.attributes.push({ key: 'exemplar_trace_id', value: trace.data.traceID });
```

Comment:
@everett980 do we support span IDs for DDG exemplars?

Reply:
@yurishkuro not yet. I asked @vprithvi about it but it wasn't a priority at the time.
Once my inflight work for the trace quality v2 page is landed, it'll be easier to add to the DDG as that page has many deep links and deep link utils.
However url length restrictions are already hit without deep linking, and deep linking would double the length so even then it won't be a super trivial change.
Let's create a ticket for it and put a pin it for now.